functional-tests/devscripts/route_tests.sh

```diff
@@ -39,8 +44,30 @@ function checkPrereq {
 		fi
 		installOC
 	fi
+	
+	yum install -y bc
+	
+	if $SEND_TO_ZABBIX; then
+		rpm -ivh https://repo.zabbix.com/zabbix/3.0/rhel/7/x86_64/zabbix-release-3.0-1.el7.noarch.rpm
+		yum install -y zabbix-sender
+	fi
 }
 
+# Setting default ZABBIX values
+SEND_TO_ZABBIX=true
+ZABBIX_SERVER="zabbix.devshift.net"
+ZABBIX_PORT="9443"
+case "$OC_CLUSTER_URL" in
+    *"2a"*) ZABBIX_HOST="qa-starter-us-east-2a"
+            ;;
+    *"1a"*) ZABBIX_HOST="qa-starter-us-east-1a"
+	        ;;
+    *"1b"*) ZABBIX_HOST="qa-starter-us-east-1b"
+            ;;
+    *) ZABBIX_HOST="qa-starter-us-east-2"
```

Comment:
IMHO this could be dangerous. I suggest to list `2` explicitly and in the case of `*` would set the host to something like `unknown`, `undefined`, `null` or something similar.